src/ims/application/_api.py

```diff
@@ -145,7 +144,6 @@ def pingResource(self, request: IRequest) -> KleinRenderable:
         return jsonBytes(request, ack, str(hash(ack)))
 
     @router.route(_unprefix(URLs.bag), methods=("HEAD", "GET"))
-    @static
```

Comment:
The bag is static with the code and shouldn't need to be reloaded unless new code was deployed…

Reply:
agreed, that one is actually static